docs/versions.md

```diff
@@ -14,5 +14,11 @@ Breaking changes are documented in the release notes and in the documentation.
 
 - See [Upgrading to version 0.10.0][2]
 
+## Ark versions and Kubernetes versions
+
+Not all Ark versions support all versions of Kubernetes. You should be aware of the following known limitations:
+
+- Ark version 0.9.0 requires Kubernetes version 1.8 or later. In version 0.9.1, Ark was updated to support earlier versions.
```

Comment:
restic support requires kube 1.10 or later (or an earlier version with the mount propagation feature gate manually enabled). This is listed in one of our docs somewhere.